packages/office-ui-fabric-react/src/components/Panel/Panel.types.ts

```diff
@@ -75,7 +77,18 @@ export interface IPanelProps extends React.Props<Panel> {
   onDismissed?: () => void;
 
   /**
-   * Additional styling options.
+   * Call to provide customized styling that will layer on top of the variant rules.
+   */
+  styles?: IStyleFunctionOrObject<IPanelStyleProps, IPanelStyles>;
```

Comment:
You're adding a property to the official documentation here, but it doesn't work. In fact i don't even see a .styles file.

Can you fix this by not changing the type exports until you actually respect them? This has caused serious confusion for end users when components are left in a halfway state.

If you absolutely need these props now, you need to be very clear in the documentation that they aren't supported but will be in a future release.

Reply:
@bennettclark Just a heads up on something to look for. If we add props, they'll get documented on a website update pretty quickly, and will confuse and frustrate users when they aren't functional :)

Reply:
Right, I do recall talking to @cliffkoh about the two-part PR being problematic, but was following the official documented method till told otherwise.

We originally separated out the PRs because the one large PR was deemed hard to review.  I think that was John Miller's call though, so we can change that.  I don't think a single PR would be bad on this component as its small/medium in size.  The main factor is most of the key changes are to the component/base file and when you restructure the files the create the Component.base.ts file it will just look like a new file and a deleted file - github will not outline the changes I made to the component in the "phase 2" and it was considered likely that important things might be missed in the review.  The in-between documentation time was considered but found to be the less problematic.

Do we want to switch to 1 big PR, or another alternative way?

Reply:
The Dropdown PR and DetailsList for instance, do not have this problem. Git and GitHub recognized the rename of Dropdown.tsx -> Dropdown.base.tsx. Important thing is that the root file has an extension of `ts`, so `Dropdown.ts` instead of `Dropdown.tsx` such that it correctly shows up as a new file.

David has a differently idea of what phase1/phase2 means (from John Miller) but I think it is easier to just do it in 1 PR honestly :)